franklin/blocks/page-feed/page-feed.js

```diff
@@ -0,0 +1,383 @@
+import {
+  createTag,
+  getConfig,
+  replaceKey,
+} from '../../scripts/scripts/scripts.js';
+
+function turnH6intoDetailM(scope) {
```

Comment:
This file needs a header and JSDoc for all these functions, I would like to see documentation on how the page feed is to be used, by example.

Reply:
Could we add a comment to include these 2 use cases in this file or a README? That way it will be much easier for someone in the future to understand the design and usage examples.

Reply:
Sure. I'm actually going to leave the link in PR description as a reference in code and update that page to contain all use cases later. I think it's probably not the best to be using a link from the old repo as documentation. Would that be OK?